hapi/src/mod.cjs

```diff
@@ -0,0 +1,6 @@
+const fetch = (...args) =>
+  import('node-fetch').then(({ default: fetch }) => fetch(...args))
+
+module.exports = {
+  fetch
+}
```

Comment:
@tetogomez please could you validate this? More on [this link](https://github.com/node-fetch/node-fetch#commonjs)

Reply:
nice!